pkg/convert/testdata/programs/heredoc_template_expr/pcl/main.pp

```diff
@@ -0,0 +1,4 @@
+aList            = ["a", "b", "c"]
+joinTemplateExpr = <<EOT
+%{for v in aList~}${v}\n%{endfor~}
+EOT
```

Comment:
Just sanity checking, sorry if dumb question -- do we want it realised this way, with the `\n` in the literal, or should we actually write out line breaks?

Reply:
I believe we want it realized this way because then it will be reparsed identically